tree/color.go

```diff
@@ -21,14 +23,5 @@ var (
 )
 
 func randomColor() *color.Color {
-	rand.Seed(time.Now().UnixNano())
 	return colors[rand.Intn(colorSize)]
 }
-
-func yellowColor() *color.Color {
```

Comment:
colors[1] can be painful if someone will change the order of colors.

Reply:
you are right.